index.html

```diff
@@ -83,6 +82,125 @@ <h2>Software Developer</h2>
           <img src="./icons/work.png" alt="image" />
         </div>
       </section>
+
+      <!-- My Recent work -->
+      <section class="works-section align-center">
+        <h2 class="align-left justify-center">My Recent Work</h2>
+        <hr />
+
+        <div class="card-container justify-center">
+          <div class="card d-flex">
+            <img
+              src="./images/works.png"
+              class="img-fluid rounded-start"
+              alt="..."
+            />
+            <div class="card-body">
+              <h3 class="card-title align-left">Multi-Post Stories</h3>
+              <p class="card-text align-left">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+
+              <div class="tags d-flex space-between">
+                <div class="align-center">css</div>
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+              </div>
+              <div class="card-button align-left">
+                <button class="align-left" type="submit">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="card-container justify-center">
+          <div class="card d-flex">
+            <img
+              src="./images/works.png"
+              class="img-fluid rounded-start"
+              alt="..."
+            />
+            <div class="card-body">
+              <h3 class="card-title align-left">Multi-Post Stories</h3>
+              <p class="card-text align-left">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+
+              <div class="tags d-flex space-between">
+                <div class="align-center">css</div>
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
```

Comment:
- [ ] This should be wrapped by `ul` tag.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li